requirements.in

```diff
@@ -0,0 +1,23 @@
+boto3
+# removed hdfs
+apache-airflow[celery,postgres,hive,jdbc,async,password,crypto,github_enterprise,datadog,statsd,s3,mysql,google_auth,gcp_api,kubernetes]==1.10.10
+mozlogging
+retrying
+newrelic
+redis
+hiredis
+requests
+jsonschema
+Flask-OAuthlib
+pytz
+werkzeug==0.16.0
+# The next requirements are for kubernetes-client/python
+urllib3>=1.24.2 # MIT
+ipaddress>=1.0.17;python_version=="2.7"  # PSF
```

Comment:
not sure what implications this one has with python version = 2.7

Reply:
It looks like ipaddress isn't included in the compiled dependencies since this is now standard library.